app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,25 @@
+<resources>
```

Comment:
All string resource names have to be given independent of their inital use case.